packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -949,23 +949,3 @@ class FlutterPluginTask extends DefaultTask {
         }
     }
 }
-
-gradle.useLogger(new FlutterEventLogger())
```

Comment:
Could we get something in the PR description explaining why this was removed?  I'm assuming it uses deprecated APIs - what's the replacement (or why do we not need a replacement)?

Reply:
Done. This was mentioned in #32550

Reply:
Done